inrupt/pod-browser
# Feat: add app profile

# App profile

For now, all the app data is mocked. 

## To test: 
- Verify that the new app profile matches the acceptance criteria
- Verify that the person profile still works as intended both in the Privacy Tab and the Contacts Tab, accessing it both from the link in the Table and in the Details Drawer

# Checklist
